docs/5.x/tracking-javascript.md

```diff
@@ -38,8 +38,7 @@ Read also the **[JavaScript Tracking Client](/guides/tracking-javascript-guide)*
 * `getCrossDomainLinkingUrlParameter()` - Get the query parameter to append to links to handle cross domain linking. Use this to add cross domain support for links that are added to the DOM dynamically.  [Learn more about cross domain linking](https://matomo.org/faq/how-to/faq_23654/). 
 * `disableBrowserFeatureDetection()` - By default, Matomo accesses information from the visitor's browser to detect the current browser resolution and what browser plugins (for example PDF and cookies) are supported. This information is used to show you reports on your visitor's browser resolution, supported browser plugins, and it is also used to generate a short-lived identifier for every visitor which we call the [config_id](https://matomo.org/faq/general/how-is-the-visitor-config_id-processed/). Some privacy regulations may only allow accessing information from a visitor's device after having consent. If this applies to you, call this method to no longer access this information. [Learn more about browser feature detection](https://matomo.org/faq/how-do-i-disable-browser-feature-detection-completely/)
 * `enableBrowserFeatureDetection()` - Enable the browser feature detection if you previously disabled it. 
-* `disableCampaignParameters()` - By default, Matomo will send campaign parameters (mtm, utm etc) to the tracker and record that information. Some privacy regulations may not allow for this information to be collected unless there is consent given. If this is used in a combination with `requireConsent` then once tracking consent has been given the tracker will start collecting this information. If it's used alone then campaign parameters will never be sent.
-* `enableCampaignParameters()` - Enable campaign parameters being sent to Matomo if you previously disabled it.
+* `disableCampaignParameters()` - By default, Matomo will send campaign parameters (mtm, utm etc) to the tracker and record that information. Some privacy regulations may not allow for this information to be collected. If this applies to you call this method to prevent campaign parameters being sent to the tracker.
```

Comment:
Minor grammar tweak on top of Stefan's suggestion.
```suggestion
* `disableCampaignParameters()` - By default, Matomo will send campaign parameters (mtm, utm, etc.) to the tracker and record that information. Some privacy regulations may not allow for this information to be collected. If this applies to you, call this method to prevent campaign parameters from being sent to the tracker. This method is available as of Matomo 5.1.
```